api/py/ai/chronon/observability/__init__.py

```
@@ -0,0 +1 @@
+__all__ = ['ttypes', 'constants']
```

Pylint (3.3.7)

[error] Undefined variable name 'ttypes' in __all__

(E0603)